package.json

```diff
@@ -3,17 +3,22 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "format": "prettier --check \"./resources/js/**/*.{js,cjs,mjs,ts,tsx,md,json}\" --ignore-path ../../.gitignore",
+    "format": "prettier --write \"./resources/js/**/*.{js,cjs,mjs,ts,tsx,md,json}\" --ignore-path ../../.gitignore",
```

Comment:
the normal format script should be just --check because it's being called in the github actions and we DON'T want top modify our code there, only locally. Please add a different script that does the --write just so we are sure we are calling it only ever by hand

Reply:
sorry I forgot to remove it I was trying that --write option, should I make another different script or just put it back as it was before

Reply:
let's make another script

Reply:
done